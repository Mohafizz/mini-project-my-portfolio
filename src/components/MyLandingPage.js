import React, { Component } from "react";
import "./MyLandingPage.css";
import { Layout, Header, Navigation, Drawer, Content } from "react-mdl";
import MySkillsRadarChart from "./MySkillsRadarChart";

class MyLandingPage extends Component {
  render() {
    return (
      <div style={{ height: "500px", position: "relative" }}>
        <Layout
          style={{
            backgroundColor: "cyan"
          }}
        >
          <Header
            transparent="false"
            title="Mohamed Hafiz"
            style={{ color: "white" }}
          >
            <Navigation>
              <a href="/">Link</a>
              <a href="/">Link</a>
              <a href="/">Link</a>
              <a href="/">Link</a>
            </Navigation>
          </Header>
          <Drawer title="My portfolio">
            <Navigation>
              <a href="/">Link</a>
              <a href="/">Link</a>
              <a href="/">Link</a>
              <a href="/">Link</a>
            </Navigation>
          </Drawer>
          <Content />
        </Layout>
        <MySkillsRadarChart />
      </div>
    );
  }
}

export default MyLandingPage;
