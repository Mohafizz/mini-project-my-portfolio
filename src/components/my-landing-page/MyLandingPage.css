.landing-header-color {
  background: #fc00ff; /* fallback for old browsers */
  background: -webkit-linear-gradient(
    to right,
    #00dbde,
    #fc00ff
  ); /* Chrome 10-25, Safari 5.1-6 */
  background: linear-gradient(
    to right,
    #00dbde,
    #fc00ff
  ); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
}

.react-rotating-text-cursor {
  animation: blinking-cursor 0.8s cubic-bezier(0.68, 0.01, 0.01, 0.99) 0s
    infinite;
}

.rotating-text {
  font-size: 2em;
  vertical-align: bottom;
}

@keyframes blinking-cursor {
  0% {
    opacity: 0;
  }
  50% {
    opacity: 1;
  }
  100% {
    opacity: 0;
  }
}

.header-image {
  width: 100%;
  height: 200px;
}
