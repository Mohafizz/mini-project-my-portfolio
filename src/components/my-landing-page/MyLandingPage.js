import React, { Component } from "react";
import { Link } from "react-router-dom";
import "react-mdl/extra/material.js";
import {
  IconButton,
  Layout,
  Header,
  Navigation,
  Drawer,
  Content
} from "react-mdl";
import AppRouter from "../app-router/AppRouter";
import "./MyLandingPage.css";
import MyFooter from "../footer/footer";
import ReactRotatingText from "react-rotating-text";

class MyLandingPage extends Component {
  render() {
    return (
      <div className="demo-big-content">
        <Layout>
          <Header
            className="landing-header-color"
            title="Mohamed Hafiz Bin Abu Bakar"
            scroll
          >
            <ReactRotatingText
              className="rotating-text"
              color="orange"
              items={["Reset...", "Relearn...", "Reprogram..."]}
            />
            <IconButton name="home" href="/" />
            <Navigation>
              <Link to="/weather-app">Weather App</Link>
              <Link to="/projects">Projects</Link>
              <Link to="/skills">Skills</Link>
            </Navigation>
          </Header>
          <Drawer title="My portfolio">
            <Navigation>
              <Link to="/">Home page</Link>
              <Link to="/weather-app">Weather App</Link>
              <Link to="/projects">Projects</Link>
              <Link to="/skills">Skills</Link>
            </Navigation>
          </Drawer>

          <Content>
            <div
              className="header-image"
              style={{
                background:
                  "url(Images/lego-cropped.jpg) right no-repeat #4BBCE3"
              }}
            />
            <AppRouter />
            <MyFooter />
          </Content>
        </Layout>
      </div>
    );
  }
}

export default MyLandingPage;
