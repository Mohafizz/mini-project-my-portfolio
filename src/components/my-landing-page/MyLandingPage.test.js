import React from "react";
import { shallow } from "enzyme";
import MyLandingPage from "./MyLandingPage";

const landingPage = shallow(<MyLandingPage />);

it("should render correctly", () => {
  expect(landingPage).toMatchSnapshot();
});
