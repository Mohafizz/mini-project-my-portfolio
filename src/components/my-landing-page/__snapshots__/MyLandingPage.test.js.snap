// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MyLandingPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Layout>
        <Header
                className="landing-header-color"
                scroll={true}
                title="Mohamed Hafiz Bin Abu Bakar"
        >
                <ReactRotatingText
                        className="rotating-text"
                        color="orange"
                        cursor={true}
                        deletingInterval={50}
                        emptyPause={1000}
                        eraseMode="erase"
                        items={
                                Array [
                                        "Reset...",
                                        "Relearn...",
                                        "Reprogram...",
                                      ]
                        }
                        pause={1500}
                        typingInterval={50}
                />
                <IconButton
                        href="/"
                        name="home"
                />
                <Navigation>
                        <Link
                                replace={false}
                                to="/weather-app"
                        >
                                Weather App
                        </Link>
                        <Link
                                replace={false}
                                to="/projects"
                        >
                                Projects
                        </Link>
                        <Link
                                replace={false}
                                to="/skills"
                        >
                                Skills
                        </Link>
                </Navigation>
        </Header>
        <Drawer
                title="My portfolio"
        >
                <Navigation>
                        <Link
                                replace={false}
                                to="/"
                        >
                                Home page
                        </Link>
                        <Link
                                replace={false}
                                to="/weather-app"
                        >
                                Weather App
                        </Link>
                        <Link
                                replace={false}
                                to="/projects"
                        >
                                Projects
                        </Link>
                        <Link
                                replace={false}
                                to="/skills"
                        >
                                Skills
                        </Link>
                </Navigation>
        </Drawer>
        <Content>
                <div
                        className="header-image"
                        style={
                                Object {
                                        "background": "url(Images/lego-cropped.jpg) right no-repeat #4BBCE3",
                                      }
                        }
                />
                <AppRouter />
                <MyFooter />
        </Content>
</Layout>,
      "className": "demo-big-content",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Header
            className="landing-header-color"
            scroll={true}
            title="Mohamed Hafiz Bin Abu Bakar"
>
            <ReactRotatingText
                        className="rotating-text"
                        color="orange"
                        cursor={true}
                        deletingInterval={50}
                        emptyPause={1000}
                        eraseMode="erase"
                        items={
                                    Array [
                                                "Reset...",
                                                "Relearn...",
                                                "Reprogram...",
                                              ]
                        }
                        pause={1500}
                        typingInterval={50}
            />
            <IconButton
                        href="/"
                        name="home"
            />
            <Navigation>
                        <Link
                                    replace={false}
                                    to="/weather-app"
                        >
                                    Weather App
                        </Link>
                        <Link
                                    replace={false}
                                    to="/projects"
                        >
                                    Projects
                        </Link>
                        <Link
                                    replace={false}
                                    to="/skills"
                        >
                                    Skills
                        </Link>
            </Navigation>
</Header>,
          <Drawer
            title="My portfolio"
>
            <Navigation>
                        <Link
                                    replace={false}
                                    to="/"
                        >
                                    Home page
                        </Link>
                        <Link
                                    replace={false}
                                    to="/weather-app"
                        >
                                    Weather App
                        </Link>
                        <Link
                                    replace={false}
                                    to="/projects"
                        >
                                    Projects
                        </Link>
                        <Link
                                    replace={false}
                                    to="/skills"
                        >
                                    Skills
                        </Link>
            </Navigation>
</Drawer>,
          <Content>
            <div
                        className="header-image"
                        style={
                                    Object {
                                                "background": "url(Images/lego-cropped.jpg) right no-repeat #4BBCE3",
                                              }
                        }
            />
            <AppRouter />
            <MyFooter />
</Content>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <ReactRotatingText
                className="rotating-text"
                color="orange"
                cursor={true}
                deletingInterval={50}
                emptyPause={1000}
                eraseMode="erase"
                items={
                                Array [
                                                "Reset...",
                                                "Relearn...",
                                                "Reprogram...",
                                              ]
                }
                pause={1500}
                typingInterval={50}
/>,
              <IconButton
                href="/"
                name="home"
/>,
              <Navigation>
                <Link
                                replace={false}
                                to="/weather-app"
                >
                                Weather App
                </Link>
                <Link
                                replace={false}
                                to="/projects"
                >
                                Projects
                </Link>
                <Link
                                replace={false}
                                to="/skills"
                >
                                Skills
                </Link>
</Navigation>,
            ],
            "className": "landing-header-color",
            "scroll": true,
            "title": "Mohamed Hafiz Bin Abu Bakar",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "className": "rotating-text",
                "color": "orange",
                "cursor": true,
                "deletingInterval": 50,
                "emptyPause": 1000,
                "eraseMode": "erase",
                "items": Array [
                  "Reset...",
                  "Relearn...",
                  "Reprogram...",
                ],
                "pause": 1500,
                "typingInterval": 50,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "href": "/",
                "name": "home",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  <Link
                    replace={false}
                    to="/weather-app"
>
                    Weather App
</Link>,
                  <Link
                    replace={false}
                    to="/projects"
>
                    Projects
</Link>,
                  <Link
                    replace={false}
                    to="/skills"
>
                    Skills
</Link>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Weather App",
                    "replace": false,
                    "to": "/weather-app",
                  },
                  "ref": null,
                  "rendered": "Weather App",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Projects",
                    "replace": false,
                    "to": "/projects",
                  },
                  "ref": null,
                  "rendered": "Projects",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Skills",
                    "replace": false,
                    "to": "/skills",
                  },
                  "ref": null,
                  "rendered": "Skills",
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Navigation>
              <Link
                            replace={false}
                            to="/"
              >
                            Home page
              </Link>
              <Link
                            replace={false}
                            to="/weather-app"
              >
                            Weather App
              </Link>
              <Link
                            replace={false}
                            to="/projects"
              >
                            Projects
              </Link>
              <Link
                            replace={false}
                            to="/skills"
              >
                            Skills
              </Link>
</Navigation>,
            "title": "My portfolio",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <Link
                  replace={false}
                  to="/"
>
                  Home page
</Link>,
                <Link
                  replace={false}
                  to="/weather-app"
>
                  Weather App
</Link>,
                <Link
                  replace={false}
                  to="/projects"
>
                  Projects
</Link>,
                <Link
                  replace={false}
                  to="/skills"
>
                  Skills
</Link>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Home page",
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": "Home page",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Weather App",
                  "replace": false,
                  "to": "/weather-app",
                },
                "ref": null,
                "rendered": "Weather App",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Projects",
                  "replace": false,
                  "to": "/projects",
                },
                "ref": null,
                "rendered": "Projects",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Skills",
                  "replace": false,
                  "to": "/skills",
                },
                "ref": null,
                "rendered": "Skills",
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <div
                className="header-image"
                style={
                                Object {
                                                "background": "url(Images/lego-cropped.jpg) right no-repeat #4BBCE3",
                                              }
                }
/>,
              <AppRouter />,
              <MyFooter />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "header-image",
                "style": Object {
                  "background": "url(Images/lego-cropped.jpg) right no-repeat #4BBCE3",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Layout>
          <Header
                    className="landing-header-color"
                    scroll={true}
                    title="Mohamed Hafiz Bin Abu Bakar"
          >
                    <ReactRotatingText
                              className="rotating-text"
                              color="orange"
                              cursor={true}
                              deletingInterval={50}
                              emptyPause={1000}
                              eraseMode="erase"
                              items={
                                        Array [
                                                  "Reset...",
                                                  "Relearn...",
                                                  "Reprogram...",
                                                ]
                              }
                              pause={1500}
                              typingInterval={50}
                    />
                    <IconButton
                              href="/"
                              name="home"
                    />
                    <Navigation>
                              <Link
                                        replace={false}
                                        to="/weather-app"
                              >
                                        Weather App
                              </Link>
                              <Link
                                        replace={false}
                                        to="/projects"
                              >
                                        Projects
                              </Link>
                              <Link
                                        replace={false}
                                        to="/skills"
                              >
                                        Skills
                              </Link>
                    </Navigation>
          </Header>
          <Drawer
                    title="My portfolio"
          >
                    <Navigation>
                              <Link
                                        replace={false}
                                        to="/"
                              >
                                        Home page
                              </Link>
                              <Link
                                        replace={false}
                                        to="/weather-app"
                              >
                                        Weather App
                              </Link>
                              <Link
                                        replace={false}
                                        to="/projects"
                              >
                                        Projects
                              </Link>
                              <Link
                                        replace={false}
                                        to="/skills"
                              >
                                        Skills
                              </Link>
                    </Navigation>
          </Drawer>
          <Content>
                    <div
                              className="header-image"
                              style={
                                        Object {
                                                  "background": "url(Images/lego-cropped.jpg) right no-repeat #4BBCE3",
                                                }
                              }
                    />
                    <AppRouter />
                    <MyFooter />
          </Content>
</Layout>,
        "className": "demo-big-content",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Header
              className="landing-header-color"
              scroll={true}
              title="Mohamed Hafiz Bin Abu Bakar"
>
              <ReactRotatingText
                            className="rotating-text"
                            color="orange"
                            cursor={true}
                            deletingInterval={50}
                            emptyPause={1000}
                            eraseMode="erase"
                            items={
                                          Array [
                                                        "Reset...",
                                                        "Relearn...",
                                                        "Reprogram...",
                                                      ]
                            }
                            pause={1500}
                            typingInterval={50}
              />
              <IconButton
                            href="/"
                            name="home"
              />
              <Navigation>
                            <Link
                                          replace={false}
                                          to="/weather-app"
                            >
                                          Weather App
                            </Link>
                            <Link
                                          replace={false}
                                          to="/projects"
                            >
                                          Projects
                            </Link>
                            <Link
                                          replace={false}
                                          to="/skills"
                            >
                                          Skills
                            </Link>
              </Navigation>
</Header>,
            <Drawer
              title="My portfolio"
>
              <Navigation>
                            <Link
                                          replace={false}
                                          to="/"
                            >
                                          Home page
                            </Link>
                            <Link
                                          replace={false}
                                          to="/weather-app"
                            >
                                          Weather App
                            </Link>
                            <Link
                                          replace={false}
                                          to="/projects"
                            >
                                          Projects
                            </Link>
                            <Link
                                          replace={false}
                                          to="/skills"
                            >
                                          Skills
                            </Link>
              </Navigation>
</Drawer>,
            <Content>
              <div
                            className="header-image"
                            style={
                                          Object {
                                                        "background": "url(Images/lego-cropped.jpg) right no-repeat #4BBCE3",
                                                      }
                            }
              />
              <AppRouter />
              <MyFooter />
</Content>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <ReactRotatingText
                  className="rotating-text"
                  color="orange"
                  cursor={true}
                  deletingInterval={50}
                  emptyPause={1000}
                  eraseMode="erase"
                  items={
                                    Array [
                                                      "Reset...",
                                                      "Relearn...",
                                                      "Reprogram...",
                                                    ]
                  }
                  pause={1500}
                  typingInterval={50}
/>,
                <IconButton
                  href="/"
                  name="home"
/>,
                <Navigation>
                  <Link
                                    replace={false}
                                    to="/weather-app"
                  >
                                    Weather App
                  </Link>
                  <Link
                                    replace={false}
                                    to="/projects"
                  >
                                    Projects
                  </Link>
                  <Link
                                    replace={false}
                                    to="/skills"
                  >
                                    Skills
                  </Link>
</Navigation>,
              ],
              "className": "landing-header-color",
              "scroll": true,
              "title": "Mohamed Hafiz Bin Abu Bakar",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "className": "rotating-text",
                  "color": "orange",
                  "cursor": true,
                  "deletingInterval": 50,
                  "emptyPause": 1000,
                  "eraseMode": "erase",
                  "items": Array [
                    "Reset...",
                    "Relearn...",
                    "Reprogram...",
                  ],
                  "pause": 1500,
                  "typingInterval": 50,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "href": "/",
                  "name": "home",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": Array [
                    <Link
                      replace={false}
                      to="/weather-app"
>
                      Weather App
</Link>,
                    <Link
                      replace={false}
                      to="/projects"
>
                      Projects
</Link>,
                    <Link
                      replace={false}
                      to="/skills"
>
                      Skills
</Link>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Weather App",
                      "replace": false,
                      "to": "/weather-app",
                    },
                    "ref": null,
                    "rendered": "Weather App",
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Projects",
                      "replace": false,
                      "to": "/projects",
                    },
                    "ref": null,
                    "rendered": "Projects",
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Skills",
                      "replace": false,
                      "to": "/skills",
                    },
                    "ref": null,
                    "rendered": "Skills",
                    "type": [Function],
                  },
                ],
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <Navigation>
                <Link
                                replace={false}
                                to="/"
                >
                                Home page
                </Link>
                <Link
                                replace={false}
                                to="/weather-app"
                >
                                Weather App
                </Link>
                <Link
                                replace={false}
                                to="/projects"
                >
                                Projects
                </Link>
                <Link
                                replace={false}
                                to="/skills"
                >
                                Skills
                </Link>
</Navigation>,
              "title": "My portfolio",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  <Link
                    replace={false}
                    to="/"
>
                    Home page
</Link>,
                  <Link
                    replace={false}
                    to="/weather-app"
>
                    Weather App
</Link>,
                  <Link
                    replace={false}
                    to="/projects"
>
                    Projects
</Link>,
                  <Link
                    replace={false}
                    to="/skills"
>
                    Skills
</Link>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Home page",
                    "replace": false,
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": "Home page",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Weather App",
                    "replace": false,
                    "to": "/weather-app",
                  },
                  "ref": null,
                  "rendered": "Weather App",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Projects",
                    "replace": false,
                    "to": "/projects",
                  },
                  "ref": null,
                  "rendered": "Projects",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Skills",
                    "replace": false,
                    "to": "/skills",
                  },
                  "ref": null,
                  "rendered": "Skills",
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <div
                  className="header-image"
                  style={
                                    Object {
                                                      "background": "url(Images/lego-cropped.jpg) right no-repeat #4BBCE3",
                                                    }
                  }
/>,
                <AppRouter />,
                <MyFooter />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "header-image",
                  "style": Object {
                    "background": "url(Images/lego-cropped.jpg) right no-repeat #4BBCE3",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
